modules/ftm/src/visitoruploadcrawler.cpp

```diff
@@ -20,13 +20,11 @@ const std::string VisitorUploadCrawler::exIntro = "VisitorUploadCrawler: ";
 //FIXME Dear future self, the code duplication in here is appalling. Create some proper helper functions for crying out loud !
 
 void VisitorUploadCrawler::visit(const Block& el) const {
-  el.serialise(getDefDst() + getQInfo(), b);
-  getRefLinks();
+  el.serialise(getDefDst() + getQInfo() + getRefLinks(), b);
 }
 
 void VisitorUploadCrawler::visit(const TimingMsg& el) const  {
-  el.serialise(getDefDst() + getDynSrc(), b);
-  getRefLinks();
+  el.serialise(getDefDst() + getRefLinks(), b);
```

Comment:
should be el.serialise(getDefDst() + getDynSrc() + getRefLinks(), b);

Reply:
Correct for legacy dynpar support, approved